src/components/text-editor/prosemirror-adapter/plugins/image/node.ts

```
@@ -96,12 +119,15 @@ function createImageNodeSpec(language: Languages): NodeSpec {
-            state: { default: '' },
-        },
+            state: { default: 'success' },
+        } as ImageNodeAttrSpec,
         toDOM: (node): DOMOutputSpec => {
-            if (node.attrs.state === ImageState.FAILED) {
+            if (node.attrs.state === 'idontexist') {
                 return createStatusSpan('failed', node, language);
-            } else if (node.attrs.state === ImageState.LOADING) {
+            } else if (
+                node.attrs.state === 'loading' &&
+                typeof node.attrs.state === 'EditorImageState'
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: "editorimagestate" is not a valid type name

not a valid type name

(lint/suspicious/useValidTypeof)